### PR TITLE
Allow user to customise number of search results per page

### PR DESCRIPTION
**Pull request by arkerjames**
New pagination options on Preferences page for objects

- Scene
- Image
- Actor
- Studio
- Movie

All settings default to 24

Validation to prevent anything other than a positive integer being entered. Entering invalid values directly into localStorage does not cause any ill-effects.

### CODE REVIEW

File: app/src/mixins/scene.ts [hunk 1]
```diff
@@ -34,6 +34,10 @@ export default class SceneMixin extends Vue {
     return contextModule.sceneAspectRatio;
   }
 
+  get scenesPerPage() {
+    return contextModule.scenesPerPage;
+  }
+
```
Comment (leadwolf) [inline app/src/mixins/scene.ts:40]: I can't find where this is used

```suggestion
```

Comment (arkerjames) [reply app/src/mixins/scene.ts:40]: You're probably correct that it's not required. I think I left it in from some earlier experimenting. 

Thanks for reviewing it.

Comment (arkerjames) [reply app/src/mixins/scene.ts:40]: Ugh, I'm struggling a bit with GitHub tbh. 

Do I need to make any amendments to the pull request, or do you make that change?

Comment (leadwolf) [reply app/src/mixins/scene.ts:40]: You just push to your branch and the pull request will update by itself

Comment (arkerjames) [reply app/src/mixins/scene.ts]: Thanks very much, I really appreciate your help.

**Comment by leadwolf** [comment]
@arkerjames you should create separate branches and separate PRs for the search results & image flags

**Comment by arkerjames** [comment]
> @arkerjames you should create separate branches and separate PRs for the search results counts & image flags

@leadwolf  Ah crap, I didn't realise I'd done that. Any tips on rolling this back?

**Comment by leadwolf** [comment]
https://stackoverflow.com/a/21718540 This one looks simple. Worst case, you can reset to 403eb7cfb7f5da7b03a3ccf658554f2c1b2cb97d and then force push

**Comment by arkerjames** [comment]
Appreciated. I'll give it a go.

**Comment by leadwolf** [comment]
@arkerjames If you still want this PR to go through, the "Option to exclude autogenerated images from search" commit shouldn't be a part of it.
Also, you have to pull the dev branch and fix the conflicts

**Pull request closed without merging**